openff/toolkit/topology/molecule.py

```diff
@@ -4743,6 +4744,14 @@ def from_file(
                 file_format = file_path.split(".")[-1]
         file_format = file_format.upper()
 
+        if file_format == "XYZ":
+            raise UnsupportedFileTypeError(
+                "Parsing `.xyz` files is not currently supported because they lack sufficient "
+                "chemical information to be used with SMIRNOFF force fields. For more information, "
+                "see https://open-forcefield-toolkit.readthedocs.io/en/latest/faq.html or to provide "
+                "feedback please visit https://github.com/openforcefield/openff-toolkit/issues/1145."
```

Comment:
Directly linking to an issue is probably to aggressive, though I've seen some other projects do this in other contextx (i.e. issue pypa/pip issue 9617). It would be great to in general turn these "we don't support X" points into something more concrete, if only to get a sense of how commonly an unsupported behavior is reached.

Reply:
I think that directly linking an issue is fine here. It's a living topic and, as you say, it's likely to change in the short term in a way that could use feedback.